go/service/rekey.go

```diff
@@ -111,7 +117,66 @@ func (h *RekeyHandler) DebugShowRekeyStatus(ctx context.Context, sessionID int)
 }
 
 func (h *RekeyHandler) RekeyStatusFinish(ctx context.Context, sessionID int) (keybase1.Outcome, error) {
-	return h.gregor.RekeyStatusFinish(ctx, sessionID)
+	outcome, err := h.gregor.RekeyStatusFinish(ctx, sessionID)
+	if err == nil {
+		// recheck rekey status 24h from now
+		h.recheckMu.Lock()
+		h.recheckDeadline = time.Now().Add(24 * time.Hour)
+		h.recheckMu.Unlock()
+	}
+	return outcome, err
+}
+
+func (h *RekeyHandler) recheckRekeyStatusPeriodic() {
+	h.G().Log.Debug("starting recheck rekey status loop")
+	ticker := time.NewTicker(1 * time.Hour)
+	h.G().PushShutdownHook(func() error {
+		h.G().Log.Debug("stopping recheckRekeyStatus timer")
+		ticker.Stop()
+		return nil
+	})
+	go func() {
+		for {
```

Comment:
It seemed simpler to have this running and do a comparison against zero every hour.  Then setting a deadline every time they cancel a rekey window.